src/index.html

```diff
@@ -6,13 +6,73 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="prodact image"
+        class="card__image"
+      />
+
+      <div class="card__prodact-name">
+        <h2>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      </div>
```

Comment:
```
<h2 class="card__prodact-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
```

Reply:
`product__name`, rename the block from `card` to `product` everywhere, cause it make more sense

Reply:
Make sure you fix it everywhere